frontend/integration-tests/views/catalog.view.ts

```diff
@@ -1,7 +1,8 @@
 import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
 
 export const categoryTabs = $$('.vertical-tabs-pf-tab > a');
-export const categoryTabsPresent = () => browser.wait(until.presenceOf($('.vertical-tabs-pf-tab')));
+export const categoryTabsPresent = () =>
+  browser.wait(until.presenceOf($$('.vertical-tabs-pf-tab').first()));
```

Comment:
Why was this change needed?

Reply:
This seems to help, but only OLM scenarios which I gave up addressing at this time, thus dropped.